src/expr/src/relation/func.rs

```diff
@@ -308,10 +308,10 @@ where
     let mut left_datum_vec = mz_repr::DatumVec::new();
     let mut right_datum_vec = mz_repr::DatumVec::new();
     let mut sort_by = |left: &(_, Row), right: &(_, Row)| {
-        let left = &left.1;
-        let right = &right.1;
-        let left_datums = left_datum_vec.borrow_with(left);
-        let right_datums = right_datum_vec.borrow_with(right);
+        let left_order_by = &left.1;
+        let right_order_by = &right.1;
+        let left_datums = left_datum_vec.borrow_with(left_order_by);
+        let right_datums = right_datum_vec.borrow_with(right_order_by);
```

Comment:
Maybe skip the binding altogether?
```suggestion
        let left_datums = left_datum_vec.borrow_with(&left.1);
        let right_datums = right_datum_vec.borrow_with(&right.1);
```

Reply:
Thanks, changed it.